lib/TextRoles/WrapperTextRole.php

```diff
@@ -29,6 +30,6 @@ public function getName(): string
 
     public function process(string $text): string
     {
-        return sprintf($this->wrap, $text);
+        return sprintf($this->wrap, htmlspecialchars($text));
```

Comment:
Sorry for my ignorance, from your commit message I understand that:
1. When `SpanProcessor` is used, Twig is used.
2. When `WrapperTextRole` is used, Twig is not used.

It's just a deduction though, and I'd like to understand a bit more why that's the case.

Reply:
The WrapperTextRole is currently used in a way that you wrap the content with an html tag. Up until now there was no escaping whatsover, so the danger of XSS.

There is currently no twig template involved in the wrapper textrole. I have however considered it might be better to use a twig template in such situations as well. This would be a follow up then

Reply:
See for example [tests/Functional/tests/render/code-role/code-role.html](https://github.com/doctrine/rst-parser/pull/245/files#diff-9178f76996b18b66b29363fa197c405d78999da36523d0929a540d61bb52986c) where you can see that output was not escaped until now